index.js

```diff
@@ -128,14 +144,17 @@ class Ky {
 		this._options.prefixUrl = String(this._options.prefixUrl || '');
 		this._input = String(input || '');
 
-		if (this._options.prefixUrl && this._input.startsWith('/')) {
-			throw new Error('`input` must not begin with a slash when using `prefixUrl`');
-		}
-		if (this._options.prefixUrl && !this._options.prefixUrl.endsWith('/')) {
-			this._options.prefixUrl += '/';
+		const isFullUrlInput = isAbsoluteUrl(this._input);
```

Comment:
I found this whole part a bit tricky. When do you consider the input relative? Some examples:

- `input` starts with `/`. This is relative.
- `input` starts with `/+`, what now? I guess it's relative and could be accepted.
- `input` starts with `[a-z]`. At this point you have to start thinking about the `input` being a full url including a protocol. How to handle this?